apps/hello/main.py

```diff
@@ -80,6 +80,75 @@ def _get_story_text():
     with open("assets/story.txt", "r") as f:
         return f.read()
 
+def chat_handler(payload, state):
+    if payload == "pdf":
+        return {
+            "text": "In this demo you can find only this PDF file.",
+            "actions": [{
+                "subheading": "Resource",
+                "name": "Neon Feathers",
+                "desc": "Click to be open",
+                "data": "open_pdf" 
+            }]
+        }
+    elif payload == "web":
+        return {
+            "text": "In this demo you can find only this web link.",
+            "actions": [{
+                "subheading": "Resource",
+                "name": "Streamsync",
+                "desc": "Click to be open",
+                "data": "open_web" 
+            }]
+        }
+
+    elif payload == "highlight" and state["chat_bot"]["show_pdf"]:
+        state["chat_bot"]["pdf"]["highlights"] = ["FeatherByte", "SynthoCorp"];
+        return "I have highlighted some interesting parts of the story."
+    elif payload == "help":
+        return "You can type `pdf` or `web` to see what this components can do."
+    else:
+        return "I don't understand that command. Type 'help' to see what is possible"
+
+def _show(name, resource, state):
```

Comment:
Can you change for a more descriptive method name, e.g. `_show_chatbot_resource`?